src/test/java/org/jenkinsci/plugins/enhancedoldbuilddiscarder/EnhancedOldBuildDiscarderTest.java

```diff
@@ -163,6 +163,39 @@ public void testPerformHoldMaxBuildsQuantityOnly() throws Exception {
 		assertEquals(1, buildList.size());
 	}
 
+	@Test
+	public void testPerformHoldMaxBuildsZeroHoldQuantity() throws Exception {
+		// testing for circumstance where max builds is 0 and days to keep is 1
+		// causing deletion of no builds except since they are all too young
```

Comment:
The sentence in the comment is missing a word or has one word too many... `except since`

Reply:
Fixed!